### PR TITLE
Preview app open and close

### PR DESCRIPTION
**Pull request by benrudolph**
@wpride @biyeun  this allows a very basic hide and unhide functionality for the preview app

cc: @czue

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 12 errors:

* corehq/apps/app_manager/static/app_manager/js/preview_app.js, line 3 - 'previewUrl' is defined but never used (no-unused-vars)
* corehq/apps/app_manager/static/app_manager/js/preview_app.js, line 6 - 'e' is defined but never used (no-unused-vars)
* corehq/apps/app_manager/static/app_manager/js/preview_app.js, line 8 - Missing semicolon. (semi)
* corehq/apps/app_manager/static/app_manager/js/preview_app.js, line 12 - Missing trailing comma. (comma-dangle)
* corehq/apps/app_manager/static/app_manager/js/preview_app.js, line 13 - Missing semicolon. (semi)
* corehq/apps/app_manager/static/app_manager/js/preview_app.js, line 14 - Missing semicolon. (semi)
* corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/navigation/navigation_view.js, line 1 - 'FormplayerFrontend' is not defined. (no-undef)
* corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/navigation/navigation_view.js, line 24 - Missing semicolon. (semi)
* corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/navigation/navigation_view.js, line 25 - Missing semicolon. (semi)
* corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/util/util.js, line 119 - Expected indentation of 8 space characters but found 6. (indent)
* corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/util/util.js, line 120 - Expected indentation of 8 space characters but found 6. (indent)
* corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/util/util.js, line 121 - Expected indentation of 4 space characters but found 2. (indent)

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ FormplayerFrontend.on("before:start", function () {
         regions: {
             main: "#menu-region",
             breadcrumb: "#breadcrumb-region",
-            caseTileStyle: "#case-tile-style",
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: @wpride this doesn't seem to be needed anymore?

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js [hunk 2]
```diff
@@ -144,8 +144,12 @@ FormplayerFrontend.on("start", function (options) {
         Backbone.history.start();
         // will be the same for every domain. TODO: get domain/username/pass from django
         if (this.getCurrentRoute() === "") {
-            if (options.previewMode) {
-                FormplayerFrontend.trigger("app:preview", options.apps[0]['_id']);
+            if (options.phoneMode) {
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: trying to be smart with naming. phoneMode should get you to view the app in the single app mode. preview mode should work in both phone and regular mode

File: corehq/apps/cloudcare/templates/formplayer/navigation.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<script id="formplayer-phone-navigation-template" type="text/template">
+    <a class="formplayer-back" href="">Back</a>
+    <a class="formplayer-reload" href="">Reload</a>
+</script>
```
Comment (benrudolph) [inline corehq/apps/cloudcare/templates/formplayer/navigation.html]: intentionally very basic until we know how we want to design it

File: corehq/apps/app_manager/static/app_manager/js/preview_app.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+hqDefine('app_manager/js/preview_app.js', function() {
+
+    var initializePreviewButton = function() {
+        var $previewBtn = $('.preview-app-btn'),
+            $previewPhoneContainer = $('.preview-phone-container');
+        $previewBtn.click(function(e) {
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/preview_app.js]: 'e' is defined but never used (no-unused-vars)

Comment (benrudolph) [inline corehq/apps/app_manager/static/app_manager/js/preview_app.js]: (ignoring, i like defining the event variable)

**Comment by biyeun** [comment]
merging so i can build on this. looks good

**Pull request merged**